app/presenters/html_publication_presenter.rb

```diff
@@ -13,6 +13,18 @@ def contents
     content_item["details"]["headings"].html_safe
   end
 
+  def isbn
+    content_item["details"]["isbn"] if content_item["details"]["isbn"].present?
+  end
+
+  def print_meta_data_contact_address
+    content_item["details"]["print_meta_data_contact_address"] if content_item["details"]["print_meta_data_contact_address"].present?
+  end
+
+  def copyright_year
+    content_item["details"]["public_timestamp"].to_date.year if content_item["details"]["public_timestamp"].present?
```

Comment:
There are a couple of things calling `content_item["details"]["public_timestamp"]`, can that be put into it's own private method? eg `public_timestamp`